src/sentry/static/sentry/app/views/performance/traceDetails/transactionGroup.tsx

```diff
@@ -32,33 +36,37 @@ class TransactionGroup extends React.Component<Props, State> {
   };
 
   render() {
-    const {continuingDepths, isLast, transaction} = this.props;
+    const {index, continuingDepths, isLast, transaction, traceInfo} = this.props;
     const {isExpanded} = this.state;
     const {children} = transaction;
 
     return (
       <React.Fragment>
         <TransactionBar
+          index={index}
           transaction={transaction}
+          traceInfo={traceInfo}
           continuingDepths={continuingDepths}
           isLast={isLast}
           isExpanded={isExpanded}
           toggleExpandedState={this.toggleExpandedState}
         />
         {isExpanded &&
-          children.map((child, index) => {
-            const isLastChild = index === children.length - 1;
+          children.map((child, idx) => {
+            const isLastChild = idx === children.length - 1;
             const hasChildren = child.children.length > 0;
 
             const newContinuingDepths =
               !isLastChild && hasChildren
                 ? [...continuingDepths, transaction.generation]
                 : [...continuingDepths];
 
+            // TODO(tonyx): figure out the index
```

Comment:
?

Reply:
This is a TODO item to get the alternating gray backgrounds. Because of the way I recursively generate the groups, its not trivial to label them in order. So I left it a TODO item